kotlin/change-viewpoint/src/main/java/com/esri/arcgisruntime/sample/changeviewpoint/MainActivity.kt

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2020 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgisruntime.sample.changeviewpoint
+
+import android.os.Bundle
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import com.esri.arcgisruntime.geometry.Point
+import com.esri.arcgisruntime.geometry.PointCollection
+import com.esri.arcgisruntime.geometry.Polyline
+import com.esri.arcgisruntime.geometry.SpatialReferences
+import com.esri.arcgisruntime.mapping.ArcGISMap
+import com.esri.arcgisruntime.mapping.Basemap
+import com.esri.arcgisruntime.mapping.Viewpoint
+import kotlinx.android.synthetic.main.activity_main.*
+
+class MainActivity : AppCompatActivity() {
+
+  private val scale = 5000.0
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_main)
+
+    // create a map with an imagery basemap and set it to the map view
+    mapView.map = ArcGISMap(Basemap.createImageryWithLabels())
+
+    // create point for starting location
+    val startPoint = Point(-14093.0, 6711377.0, SpatialReferences.getWebMercator())
+
+    // set viewpoint of map view to starting point and scale
+    mapView.setViewpointCenterAsync(startPoint, scale)
+  }
+
+  fun onAnimateClicked(view: View) {
```

Comment:
out of curiosity, are you required to pass in a `View` when you are using binding?

Reply:
Yeah, won't work without it. Kind of annoying the IDE highlights it as an unused parameter

Reply:
Good to know! Is it possible to set it as `_`?